dimagi/commcare-hq
# Added Enterprise Data Exports Report

## Product Description
Enterprise Dashboard:
<img width="1186" alt="image" src="https://github.com/user-attachments/assets/20f984ae-1bea-49af-9735-9c0a951e813c">

API:
<img width="601" alt="image" src="https://github.com/user-attachments/assets/b32b0761-1295-4a64-b861-8221ecece0cf">

Generated Report:
<img width="420" alt="image" src="https://github.com/user-attachments/assets/025415c4-2395-400d-a428-3c89ab378946">


## Technical Summary
Associated ticket: https://dimagi.atlassian.net/browse/SAAS-16298

## Feature Flag
No feature flag, as we're doing iterative releases for this

## Safety Assurance

### Safety story
Tested locally

### Automated test coverage

No automated tests so far. The generation logic should eventually be moved into its own functionality. That is the functionality I'd like to write tests for, but I think that is pending the approval of the iterators PR, which sets the precedent for this.

### QA Plan

TBD, but this should go through QA.
QA should verify that the following are listed on the report/API:
- Form Export
- Case Export
- Form Daily Saved Export
- Case Daily Saved Export
- Form Excel Integration
- Case Excel Integration

Nothing else should appear on the report, so its valid to test that something like PowerBI feeds do not appear here.



### Rollback instructions




### Labels & Review
